index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
```

Comment:
For the <html> element, the lang attribute must indicate the appropriate document language. Read more in http://codeguide.academy/html-css.html